### PR TITLE
Stream readme update

### PR DESCRIPTION
**Pull request by JoeOster**
# Existing Sample Changes
## Description
Updating per request from legal

### CODE REVIEW

File: third-party-programs.txt [hunk 1]
```diff
@@ -85,16 +81,14 @@ zlib License
...
-  Jean-loup Gailly        Mark Adler
-  jloup@gzip.org          madler@alumni.caltech.edu
 
 --------------------------------------------------------------------------------
 
-3.  Nbody
-
-MIT License
+3. Nbody
+   (c) 2019 Fabio Baruffa
```
Comment (mdbtucker) [inline third-party-programs.txt:88]: Did you intend to delete the word 'Copyright' here?

**Review by mdbtucker — COMMENTED** [review]
I don't really feel qualfied to review the changes to that third-party-programs.txt file.  Perhaps a suitable legal expert should review?

File: third-party-programs.txt (hunk 1 above)
Comment (JoeOster) [reply third-party-programs.txt:88]: No need to review, this was specified specifically from the legal team, merely a copy/paste replacement

Comment (mdbtucker) [reply third-party-programs.txt:88]: OK, rubber stamp approval on the way :-)

**Review by mdbtucker — DISMISSED** [review]
Approved based on Joe's comment that these changes have come from Intel legal department.

**Review by mdbtucker — APPROVED** [review]

**Pull request merged**